apps/ledger-live-desktop/tests/page/drawer/drawer.ts

```diff
@@ -30,6 +35,30 @@ export class Drawer extends Component {
     await this.addressValue(address).waitFor({ state: "visible" });
   }
 
+  @step("Verify provider is visible")
+  async providerIsVisible(account: Delegate) {
+    if (account.account === Account.NEAR_1 || account.account === Account.ATOM_1) {
```

Comment:
might be more suitable to make this check using account's currency, if that makes sense